Bottom: 10843d9898358fcdca9118d75f04ca1dcb5fbe39
Top:    10a6a401257809dfb94091149c41759dc6666376
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-27 13:35:51 +0100

drc: HACK HACK HACK


---

diff --git a/src/find.c b/src/find.c
index 1bbb1b5..7e79c30 100644
--- a/src/find.c
+++ b/src/find.c
@@ -2618,7 +2618,12 @@ IsPolygonInPolygon (PolygonTypePtr P1, PolygonTypePtr P2)
 
               line.Point1.X = v->point[0];
               line.Point1.Y = v->point[1];
-              line.Thickness = 2 * Bloat;
+              line.Thickness = Bloat;
+              /* Another Bloat is added by IsLineInPolygon, making the correct
+               * 2x Bloat. Ideally we would change it there, but doing so
+               * breaks some other DRC checks which rely on the broken behaviour
+               * in IsLineInPolygon.
+               */
               line.Clearance = 0;
               line.Flags = NoFlags ();
               for (v = v->next; v != &c->head; v = v->next)
